crypto/signer_test.go

```diff
@@ -107,7 +107,7 @@ func TestSignLogRoot(t *testing.T) {
 			t.Errorf("len(sig): %v, want > 0", got)
 		}
 		// Check that the signature is correct
-		if _, err := VerifySignedLogRoot(key.Public(), crypto.SHA256, slr); err != nil {
+		if err := Verify(key.Public(), crypto.SHA256, slr.LogRoot, slr.LogRootSignature); err != nil {
```

Comment:
Is this signature/key also to be removed (separately)?

Reply:
Yes, I wanted to just remove the signature verification in the PR, but without affecting the signing itself (including the code testing the signing).

Later PRs will remove the signing itself, as well as the key management infrastructure (which is unfortunately quite plentiful and deeply rooted!).